custom/readme_rx_printtofax.txt

```diff
@@ -1,25 +0,0 @@
-MI2 Print-to-Fax Option with signature and addendum/footer options
-
```

Comment:
It looks like this feature may actually exist in OpenEMR. The sig_pic and use_signature are set in config, but for some reason the addendum_file is not set in config. odd.